apis/duck/v1alpha1/conditions_types.go

```diff
@@ -1,203 +0,0 @@
-/*
-Copyright 2018 The Knative Authors
```

Comment:
isn't this being used in eventing? See https://github.com/knative/eventing/blob/master/pkg/reconciler/broker/controller.go#L98

Reply:
oops, missed the previous comments... discard